widgets/tools/skeletonview.h

```diff
@@ -50,17 +50,17 @@ class AbstractSkeletonModel : public QAbstractListModel {
 public:
     bool selectionProtection{false};
     bool selectionFromModel{false};
-    virtual int columnCount(const QModelIndex & parent = QModelIndex()) const override;
-    virtual QVariant headerData(int section, Qt::Orientation orientation, int role = Qt::DisplayRole) const override;
-    virtual Qt::ItemFlags flags(const QModelIndex & index) const override;
+    int columnCount(const QModelIndex & parent = QModelIndex()) const override;
+    QVariant headerData(int section, Qt::Orientation orientation, int role = Qt::DisplayRole) const override;
+    Qt::ItemFlags flags(const QModelIndex & index) const override;
     int rowCount(const QModelIndex &) const override;
 };
 
 class TreeModel : public AbstractSkeletonModel<TreeModel> {
     Q_OBJECT
     friend class AbstractSkeletonModel<TreeModel>;
     const std::vector<QString> header = {"ID", ""/*color*/, "Show", "#", "Comment", "Properties"};
-    const std::vector<Qt::ItemFlags> flagModifier = {Qt::ItemIsDropEnabled, 0, Qt::ItemIsUserCheckable, 0, Qt::ItemIsEditable, 0};
+    const std::vector<Qt::ItemFlags> flagModifier = {Qt::ItemIsDropEnabled, nullptr, Qt::ItemIsUserCheckable, nullptr, Qt::ItemIsEditable, nullptr};
```

Comment:
wut, lul

Reply:
Should have double check this one.

Reply:
https://doc.qt.io/qt-5/qflags.html#QFlags-2